Makefile

```diff
@@ -0,0 +1,21 @@
+linting: bandit black flake8 isort
```

Comment:
Just a suggestion, but in general I would use imperative mood like we do in commit messages and function names, so `lint` and `test` instead of `linting` and `testing`.